medusa/indexers/tmdb/tmdb.py

```diff
@@ -560,7 +560,7 @@ def get_last_updated_series(self, from_time, weeks=1, filter_show_list=None):
                     new_list.append(show.indexerid)
             total_updates = new_list
 
-        return total_updates
+        return list(total_updates)
```

Comment:
Why even bother converting the set to a list.  The set is iterable just like a list and the conversion isn't really necessary here.